editor/blocks/quote/style.scss

```diff
@@ -0,0 +1,24 @@
+#editor blockquote {
```

Comment:
To be clear, I think the editor should not define _any_ styles for rendering the content in the blocks.
The theme and plugins can supply styles, because they can also put those on the front end. But the editor can't put any on the real front end, so to get the correct preview, it shouldn't put any in the editor either.
As for "blocks defining their own default styles", a block is either core or added by user code, so how is it "defining its own"?